external-crates/move/crates/move-vm-runtime/src/debug.rs

```diff
@@ -59,15 +65,19 @@ impl FromStr for DebugCommand {
         if s.starts_with(PrintStack.debug_string()) {
             return Ok(PrintStack);
         }
-        if s.starts_with(Step.debug_string()) {
+        if s.starts_with(Step.debug_string())  || s.eq(StepSC.debug_string()) {
```

Comment:
You may want to consider an alternate approach where you keep the commands the same, (so no `StepSC`/`ContinueSC` and add a `fn debug_short_str(&self) -> Option<&str>` and have this return `None` for anything other than Continue/Step.

Then this can be something like `... || Step.debug_short_str().is_some_and(|short| s.seq(short)`. 

This would make things a bit cleaner possibly? Just something to consider :) I leave it up to you 🙂

Reply:
this is a nifty trick I will add to my arsenal!